SolutionInfo.cs

```diff
@@ -14,6 +16,8 @@
 
 // Review the values of the assembly attributes
 
+[assembly: CLSCompliant(false)]

```

Comment:
Looking at the removed attributes, it seems about 50/50 whether `CLSCompliant` was `true` or `false`.  What's the rationale behind setting everything to `false`?

Reply:
Setting those that are existing as true to false could have some unexpected consequences for those building extensions with languages other than C#.  (https://docs.microsoft.com/en-us/dotnet/api/system.clscompliantattribute?view=netframework-4.7.2) Not that it is a huge audience, but I'm not sure of the long-term impacts here.

@donker Were there a number of errors due to CLS issues?

Reply:
I can't remember precisely any more. But if you guys want to leave this as is then let's skip this PR. Maybe it was on my end. My only goal was to unify what we do so we have less surprises. But if it makes sense that some dlls do and others don't then it should stay as it was.